cleanlab_cli/dataset/upload_helpers.py

```diff
@@ -128,12 +128,12 @@ def validate_and_process_record(
         else:
             if col_feature_type == "datetime":
                 try:
-                    pd.to_datetime(column_value)
+                    pd.Timestamp(column_value)
                 except (ValueError, TypeError):
```

Comment:
Do we need to change the error types here? `ValueError` is valid, not sure `TypeError` can even be raised.

Reply:
Good point, were you thinking to narrow it or broaden it? I think we could just check for ValueError, but could change it to catch any Exception

Reply:
Would be nice for the scope should match exactly -- but I guess there's no harm in just catching any Exception.

Reply:
![Screen Shot 2022-07-15 at 5 25 46 PM](https://user-images.githubusercontent.com/13542329/179312961-ef075f5e-9809-4866-a0d8-39645541bc7a.png)

looks like TypeError is possible, but not sure if we'd every arrive at a scenario where we try to pass values other than string, int, None, etc.

Reply:
Okay, let's leave as is then.